reference/src/lib/store.js

```diff
@@ -349,6 +353,17 @@ export default class Store {
 			...vendors.map(({id}) => id),
 			...Array.from(selectedVendorIds));
 		this.vendorList = vendorList;
+
+		purposes.forEach((purpose) => {
```

Comment:
isn't better to use map function to apply values?

Reply:
it's mostly to match the vendorlist indexes once we get the values on the template